test/index.js

```diff
@@ -1182,6 +1182,78 @@ describe('list', function() {
 
   });
 
+  describe('take', function() {
+
+    it('returns a Nothing if applied to empty collection', function() {
```

Comment:
This isn't quite accurate since I assume `S.take(0, [])` evaluates to `Just([])`. I think we could remove this section since the `n > xs.length` case is tested in the following section.